environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/freeipa/FreeIpaCreationHandler.java

```diff
@@ -307,14 +333,28 @@ private void setUseCcm(Tunnel tunnel, CreateFreeIpaRequest createFreeIpaRequest)
         createFreeIpaRequest.setTunnel(tunnel);
     }
 
-    private List<InstanceGroupRequest> createInstanceGroupRequests(SecurityGroupRequest securityGroupRequest, FreeIpaCreationDto freeIpaCreation) {
+    private List<InstanceGroupRequest> createInstanceGroupRequests(CreateFreeIpaRequest createFreeIpaRequest,
+        SecurityGroupRequest securityGroupRequest,
+        EnvironmentDto environment,
+        boolean multiAzRequired) {
```

Comment:
Wrong indentation. The last 3 lines should probably be indented 4 more spaces to the right.